deployment/docker-compose.yml

```diff
@@ -3,8 +3,10 @@ version: '3.4'
 # Common Django template for Geonode, Celery and Celerycam services below
 x-common-django:
   &default-common-django
+  image: gep-benin_django:latest
```

Comment:
This needs to be annotated with the image name, so the dev container build can pick up the base image.